specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/SubAssessments/ListSubAssessments_example.json

```diff
@@ -0,0 +1,59 @@
+{
+	"parameters": {
+		"api-version": "2019-01-01-preview",
+		"scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23"
```

Comment:
>scope [](start = 3, length = 5)

subAssessment scope is always the parent assessment

should be represented here (?)